opensearch/src/main/java/org/opensearch/sql/opensearch/executor/OpenSearchQueryManager.java

```diff
@@ -21,7 +21,8 @@ public class OpenSearchQueryManager implements QueryManager {
 
   private final NodeClient nodeClient;
 
-  private static final String SQL_WORKER_THREAD_POOL_NAME = "sql-worker";
+  public static final String SQL_WORKER_THREAD_POOL_NAME = "sql_worker";
```

Comment:
[SQL has metrics](https://docs.aws.amazon.com/opensearch-service/latest/developerguide/managedomains-cloudwatchmetrics.html#managedomains-cloudwatchmetrics-sql) depend on sql-worker. Could u double confirm name change with not break the metrics?